src/QMCWaveFunctions/Fermion/syclSolverInverter.hpp

```diff
@@ -0,0 +1,139 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2019 QMCPACK developers.
+//
+// File developed by: Ye Luo, yeluo@anl.gov, Argonne National Laboratory
+//
+// File created by: Ye Luo, yeluo@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_SYCL_MKL_SOLVERINVERTOR_H
+#define QMCPLUSPLUS_SYCL_MKL_SOLVERINVERTOR_H
+
+#include "OhmmsPETE/OhmmsVector.h"
+#include "OhmmsPETE/OhmmsMatrix.h"
+#include "SYCL/SYCLallocator.hpp"
+#include "SYCL/syclBLAS.hpp"
+#include "SYCL/syclSolver.hpp"
+#include "QMCWaveFunctions/detail/SYCL/sycl_determinant_helper.hpp"
+
+namespace qmcplusplus
+{
+/** implements matrix inversion via cuSolverDN
+ * @tparam T_FP high precision for matrix inversion, T_FP >= T
+ */
+template<typename T_FP>
+class syclSolverInverter
```

Comment:
I think it would be nice if this followed the naming conventions for classes. Although an unfortunate precedent has be set by other solver wrapper classes.

Reply:
Let's keep it aligned as other programming models. rocSolver, cuSolver.